### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by synercoder** [SixLabors/ImageSharp#1217] [Q&A] [answered]
Hello all the people who work on ImageSharp. I want to thank all of you for your amazing work! ImageSharp is by far the best image library around and continues to becomes even more amazing and faster.

Personally I use ImageSharp in a few hobby projects (for example: https://github.com/synercoder/FileFormats.Pdf). I just wanted to say thank you and keep up the amazing work!

**Comment by synercoder** [comment, marked as answer]
Well, since I don't have a need for a license, the only right thing to do is becoming a backer on the Open Collective.

So now I am a monthly backer. Again, thanks for all the hard work. Stay amazing!

**Reply by JimBobSquarePants** [reply]
Thanks for your awesome support! This kind of feedback really is fantastic to receive. 😍
